builtin/logical/mssql/util.go

```diff
@@ -0,0 +1,60 @@
+package mssql
+
+import (
+	"fmt"
+	"strings"
+)
+
+// BuildDsn creates a DSN with some default and overriden values
+func BuildDsn(dsn string) string {
```

Comment:
In an earlier version of this module it did a little more than just set `app name`.  I'll just remove this method and if you want an `app name` you'll just have to specify it in your connection string.  I'll also show `app name` in the documentation examples.